src/features/users.ts

```diff
@@ -0,0 +1,21 @@
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { User } from '../types/User';
+import { getUsers } from '../api/users';
+
+const initialState = [] as User[];
+
+export const loadUsers = createAsyncThunk('users/fetch', async () => {
+  return getUsers();
+});
```

Comment:
Consider handling the `rejected` state of your `loadUsers` async thunk. This will allow you to better handle potential API errors and provide feedback to the user. You can do this by adding another case to your `extraReducers` that listens for `loadUsers.rejected` and updates the state accordingly.